src/ServiceControl/SetupBootstrapper.cs

```diff
@@ -41,13 +40,10 @@ public void Run(string username)
             containerBuilder.RegisterInstance(documentStore).As<IDocumentStore>().ExternallyOwned();
             containerBuilder.RegisterInstance(settings);
 
-            containerBuilder.RegisterModule<MigrationsModule>();
-
             using (documentStore)
             using (var container = containerBuilder.Build())
             using (NServiceBusFactory.Create(settings, container, null, documentStore, configuration, false))
             {
-                container.Resolve<MigrationsManager>().ApplyMigrations();
```

Comment:
I moved this to the "database update" routine because it feels like it belongs better there. YMMV.